frontend/src/lib/components/EditableField/EditableField.tsx

```diff
@@ -65,16 +66,30 @@ export function EditableField({
     saveButtonText = 'Save',
     notice,
 }: EditableFieldProps): JSX.Element {
-    const [localIsEditing, setLocalIsEditing] = useState(false)
+    const [localIsEditing, setLocalIsEditing] = useState(mode === 'edit')
     const [localTentativeValue, setLocalTentativeValue] = useState(value)
+    const inputRef = useRef<HTMLInputElement | HTMLTextAreaElement>()
+    const previousIsEditing = useRef<boolean>()
 
     useEffect(() => {
         setLocalTentativeValue(value)
     }, [value])
+
     useEffect(() => {
         setLocalIsEditing(mode === 'edit')
     }, [mode])
 
+    useEffect(() => {
+        // We always want to focus when switching to edit mode, but can't use autoFocus, because we don't want this to
+        // happen when the component is _initially_ rendered in edit mode.
+        if (inputRef.current && previousIsEditing.current === false && localIsEditing === true) {
```

Comment:
```suggestion
        if (inputRef.current && !previousIsEditing&& localIsEditing) {
```

Reply:
Good catch with `localIsEditing`, but we specifically need `previousIsEditing` to be `false` and not `undefined`, because only the former means that the component was previously rendered in view mode. The latter means that it was never rendered. Added this context to the comment above this condition